src/pages/AdminHomePage/AdminHomePage.tsx

```diff
@@ -8,8 +8,14 @@ const AdminHomePage = () => {
     <div>
       <SideMenu />
       <NavBar showSearchBar={false} />
```

Comment:
My comment is referring to the changes introduced in previous PR, I'm attaching comment here because previous PR is closed.

Our app has two main areas now: Readers and Admins. It is to be expected that they will need to render very similar components, like the NavBar, where some part are the same, with the variations. Most obvious thing to do is to control the look & function of the shared component with boolean flags, but it is also the way that leads to the code that is harder to reason about and harder to maintain. That's why boolean flags are considered code smells, you can read about here: https://martinfowler.com/bliki/FlagArgument.html

In this case it already proves hard to maintain because it missed tweaking the behaviour on the mobile phone. When library.productdock.rs/admin is opened on mobile, it still renders the search bar.

Also, this code:
```
          {showSearchBar && (
            <div className="add-book-button-wrapper">
              <AdminPanelButton />
            </div>
          )}
```
it seems odd because **showSearchBar** controls the presence of <AdminPanelButton />. It is not self evident from the code why search bar flag impacts the admin button.

Can try to find better way to to share similarities between Admin and Readers nav bars?